l10n_br_account/models/account_invoice.py

```diff
@@ -43,6 +43,7 @@
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
+    _order = "internal_number desc, id desc"
```

Comment:
Something strange happened when I use the _order property only in l10n_br_account the way we are discussing, it doesn't update the view, even in new db. 
Only in l10n_br_account_product this work. Anyone knows something about?